### ISSUE TITLE
Will the training dataset rendering script be released?

### ISSUE THREAD

**Issue by sus71** [yanqinJiang/Animate3D#10]
Dear authors,

Thank you for sharing this incredible work! Do you plan to release the rendering script for the training dataset?

**Comment by yanqinJiang** [comment]
Thanks for your interests in our work!
Sorry we don't have plans to release the rendering script. But if you meet problems when constructing rendering script, you could send me a email and I'm glad to discuss with you.
My email address is jiangyanqin2021@ia.ac.cn

**Comment by sus71** [comment]
Thanks for your reply. Could you share the evaluation dataset or just the meta information used in "For the evaluation of MV-VDM, we render multi-view images from the first frame of 128 held-out animated 3D objects and then generate multi-view videos conditioned on them." It seems the info of the 128 objects are currently missing in the repo.

**Comment by yanqinJiang** [comment]
The evaluation data contains some internal data, so we didn't release it. But feel free to evaluate our model on your data if you want to do comparison : )

**Comment by sus71** [comment]
Thank you for the reply! I still got some other questions:
1. For the metrics used in table 2 a), did you compute the scores for each generated single-view video separately and then take the mean across all views? For dynamic degree metric, from the [vbench calculation](https://github.com/Vchitect/VBench/blob/6446112aa373b20602556ead2b63aa282b849b3e/vbench/dynamic_degree.py#L84C4-L92C28), it seems the score is either 0 or 1 for a single view video, does the 0.388 means there are only ~39% of the evaluation videos contains motion? Or did you get this score in another way?
2. for the reported batch size 1024 in the paper, how do you calculate the batch size. When running the code, 1 batch occupies around ~60GB on a A100 gpu, how do you achieve 1024 on 32 A800 gpus?

**Comment by yanqinJiang** [comment]
1. a) yes, we compute the mean across all views. b) your understanding about dynamic degree value is right. Vbench computes dynamic score for video generation models in this way so we follow it.
2. The batch size refers to the number of images, i.e., 16 frames x 4 views x 1 sample per gpu x 32 gpu = 2048. There is a typo in the paper, batch size should be 2048, not 1024. Thanks for pointing it out. BTW, although we didn't manage to increase the sample size per gpu due to limited gpu memory, now you can utilize Deepspeed to achieve this (following the optimization techniques in LLM). Give it a try and I believe the sample size per gpu can at least be 2.
